bot/exts/utils/_eval_helper.py

```diff
@@ -0,0 +1,339 @@
+import re
+import urllib.parse
+import zlib
+from functools import partial
+from io import BytesIO
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import aiohttp
+from discord import Embed
+from discord.ext import commands
+from discord.ext.commands import Context
+from loguru import logger
+
+to_bytes = partial(bytes, encoding="utf-8")
+
+
+def _to_tio_string(couple: tuple) -> Any:
```

Comment:
Does this not return bytes?
```suggestion
def _to_tio_string(couple: tuple) -> bytes:
```